linchpin/cli/__init__.py

```diff
@@ -495,14 +496,15 @@ def lp_destroy(self, targets=(), run_id=None, tx_id=None):
         outputs = self._execute_action('destroy',
                                        targets,
                                        run_id=run_id,
-                                       tx_id=tx_id)
+                                       tx_id=tx_id,
+                                       vault_pass=vault_pass)
         if ('post' in self.pb_hooks) and (self.__meta__ == "CLI"):
             self.hook_state = '{0}{1}'.format('post', 'destroy')
         return outputs
 
 
-
-    def _execute_action(self, action, targets=(), run_id=None, tx_id=None):
+    def _execute_action(self, action, targets=(), run_id=None, tx_id=None,
+                        vault_pass=''):
```

Comment:
See the reference above for `do_action`. `vault_pass` should not need to be passed, use the context object instead.